### PR TITLE
Unbreak with fmtlib 6.0.0

### LINKED ISSUE #436: libfmt 6.0.0 breaks Waybar
Building Waybar master with libfmt 6.0.0 fails with the following error:
```
126 In file included from ../waybar-9999/include/factory.hpp:4,
127                  from ../waybar-9999/src/factory.cpp:1:
128 ../waybar-9999/include/modules/clock.hpp:5:10: fatal error: fmt/time.h: No such file or directory
129     5 | #include "fmt/time.h"
130       |          ^~~~~~~~
```

Full build log attached: [gui-apps:waybar-9999:20190826-191629.log](https://github.com/Alexays/Waybar/files/3542588/gui-apps.waybar-9999.20190826-191629.log)

### PR DESCRIPTION
**Pull request by jbeich**
Fixes #436

### CODE REVIEW

File: src/bar.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "bar.hpp"
-#include <spdlog/spdlog.h>
 #include "client.hpp"
 #include "factory.hpp"
+#include <spdlog/spdlog.h>
```
Comment (jbeich) [inline src/bar.cpp]: Given spdlog itself bundles fmtlib we should probably pass `-DSPDLOG_FMT_EXTERNAL` from meson.build instead to keep fmt API usage consistent.

Comment (Alexays) [reply src/bar.cpp]: Yeah we should use `SPDLOG_FMT_EXTERNAL`

Comment (Alexays) [reply src/bar.cpp]: Will do it in another PR

**Comment by Alexays** [comment]
Thanks :)

**Pull request merged**